<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetGffReadAttribute</class>
 <widget class="QWidget" name="SetGffReadAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="geneLineIdLabel">
       <property name="styleSheet">
        <string notr="true">color:#DC143C</string>
       </property>
       <property name="text">
        <string>Gene Line ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="geneLineIdEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="geneIdLabel">
       <property name="styleSheet">
        <string notr="true">color:#0000FF</string>
       </property>
       <property name="text">
        <string>Gene ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="geneIdEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="annoIdLabel">
       <property name="styleSheet">
        <string notr="true">color:#00ffff</string>
       </property>
       <property name="text">
        <string>Annotation ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="annoIdEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="CDSLineIdLabel">
       <property name="styleSheet">
        <string notr="true">color:#DEB887</string>
       </property>
       <property name="text">
        <string>CDS Line ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="CDSLineIdEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="UTR5LineIdLabel">
       <property name="styleSheet">
        <string notr="true">color:#7FFF00</string>
       </property>
       <property name="text">
        <string>5'UTR Line ID</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="UTR5LineIdEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="UTR3LineIdLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#6495ED</string>
       </property>
       <property name="text">
        <string>3'UTR Line ID</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="UTR3LineIdEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="saveBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="defaultBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="demoTextBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
